### PR TITLE
Devise Spec Verification

### PR DESCRIPTION
**Pull request by soulemanou-software**
**In this PR;**

> I checked all the validation

### CODE REVIEW

**Review by zakiyousufi — APPROVED** [review]
Your prject is complited now 
Solve the conflects and mergeit
Approved

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -23,6 +23,16 @@
     <%= render 'header/header' %> 
     <section>
       <%= yield %>
+        <p class="navbar-text float-right">
```
Comment (Meltrust) [inline app/views/layouts/application.html.erb:26]: _note:  The PR is in the `devise` branch. However, the review and review comments are based on the `dev` branch, as requested by the student._

# Backend review

- [ ]  You installed Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

- [ ] Your app has no tests, kindly add some in order to ensure your app's integrity.

- [ ] Your backend project does not contain a readme file.  This is a hard requirement for all Microverse projects.  Therefore, kindly go [here](https://github.com/microverseinc/readme-template) and use this template.

Kindly remember that all sections are required.

Comment (Meltrust) [inline app/views/layouts/application.html.erb:26]: # Frontend review

- [ ] I couldn't find any link or reference to the frontend project.  In your next review request, kindly add a link to the frontend project on your backend readme file so a proper review can be carried out.

**Review by Meltrust — CHANGES_REQUESTED** [review]
_note: The backend PR is in the devise branch. However, the review and review comments are based on the dev branch, as requested by the student._


Hi team,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!











#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

**Pull request closed without merging**